test/libsolidity/semanticTests/enums/enum_explicit_overflow.sol

```diff
@@ -9,12 +9,12 @@ contract test {
     }
 
     function getChoiceFromSigned(int256 x) public returns (uint256 d) {
-        choice = ActionChoices(x);
+        choice = ActionChoices(type(uint).max);
```

Comment:
Does `x` result in a syntax error after your change? If not then I'd leave `x` here because otherwise the value is no longer signed like the name says.

And if it does - why?

Reply:
Fixed. Actually that was an unnecessary change.